action.yml

```diff
@@ -0,0 +1,9 @@
+name: "Black"
+description: "The uncompromising Python code formatter."
+author: "Łukasz Langa et al"
```

Comment:
NIT: Possible messed up copy and paste?

Reply:
[Łukasz Langa](https://github.com/psf/black/blob/master/LICENSE#L3)
et al: ["And others"](https://en.wiktionary.org/wiki/et_al.#English)

Reply:
Fair enough, good example of today I learned!

Reply:
"and contributors to Black" is used in the docs:

https://github.com/psf/black/blob/9ed2542e938c327a53c17f8932ee5fc53776ba31/docs/conf.py#L83